api/src/main/java/org/openmrs/module/ModuleClassLoader.java

```diff
@@ -405,6 +408,41 @@ static boolean shouldResourceBeIncluded(Module module, URL fileUrl, String openm
 		return include;
 	}
 	
+	static boolean isMatchingConditionalResource(Module module, URL fileUrl, ModuleConditionalResource conditionalResource) {
```

Comment:
Did you intentionally decide not to make this method private?

Reply:
Yes it was intentional, I think it was important to unit test this method separately from `shouldResourceBeIncluded` to make sure it is returning true/false based on the pattern matching rather than the logic of `shouldResourceBeIncluded`. I made it package-private for this reason.